lib/deploy.js

```diff
@@ -58,6 +58,21 @@ function parseEnv(src) {
   }
   return res;
 }
+function createVolumes(container, funcs, volumes) {
```

Comment:
this function is a bit confusing, it's appending the volumes to the function and the container but it seems that the paremeter are unrelated (and not sure why it's called created?)

If you want to do something like this, my recomendation is to return an object here instead `function parseVolumes(volumes) => {volumes, volumeMounts}` so the caller can use the result and attach them to the proper containers or deployments

Reply:
changing the structure of the definition removed the need for a parse function as the attributes can be directly applied, thanks